NitroxModel/DataStructures/GameLogic/Entity.cs

```diff
@@ -18,42 +18,46 @@ public class Entity
         public Quaternion Rotation { get; set; }
 
         [ProtoMember(3)]
-        public TechType TechType { get; set; }
+        public Vector3 Scale { get; set; }
```

Comment:
You'll need to increment the world save version since this is a breaking contract change.